articles/client-platforms/react/06-calling-apis.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Calling APIs
+description: This tutorial will show you how to make authenticated api calls with ReactJS.
+---
+
+<%= include('../../_includes/_github', {
+  link: 'https://github.com/auth0-samples/auth0-react-sample/tree/master/05-Calling-Api',
+}) %>_
+
+The reason for implementing authentication in the first place is to protect information. In this case your information is a resource served from a server of any sort. Auth0 provides a squad of tools to assist you with end-to-end authentication in an application. Auth0 suggests you conform to RFC standard by sending the token through Authorization header.
+
+## 1. Add Authentication Header to Requests
+
+In order to make an authorized request in our example, we need to send an `Authorization` header that will decoded in the server to validate the authencaticated user. A good way to provide this is updating `AuthService` to wrap the [`fetch`](https://fetch.spec.whatwg.org/) and always send the correct headers:
```

Comment:
The first time I read this I did not fully understand what the idea was. 
What do you think about: "...provide this is updating `AuthService` creating a method to wrap the [`fetch`](https://fetch.spec.whatwg.org/) adding the authorization headers:"